App crash while media activity has spinner showing and user presses back button

Steps to reproduce:
* Open the app, login if not logged in
* Tap **Media** item to go to that activity
* While spinner is showing, press the back button.

Using develop branch.

Reproduced on Android emulator X86 API 15 and Nexus 6P (API 23)

```
         AndroidRuntime  E  FATAL EXCEPTION: main
                         E  java.lang.RuntimeException: Unable to resume activity {org.wordpress.android/org.wordpress.android.ui.main.WPMainActivity}: de.greenrobot.event.EventBusException: Subscriber class org.wordpress.android.ui.main.MeFragment already registered to event class org.wordpress.android.ui.prefs.PrefsEvents$MyProfileDetailsChanged
                         E      at android.app.ActivityThread.performResumeActivity(ActivityThread.java:2444)
                         E      at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:2472)
                         E      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1173)
                         E      at android.os.Handler.dispatchMessage(Handler.java:99)
                         E      at android.os.Looper.loop(Looper.java:137)
                         E      at android.app.ActivityThread.main(ActivityThread.java:4424)
                         E      at java.lang.reflect.Method.invokeNative(Native Method)
                         E      at java.lang.reflect.Method.invoke(Method.java:511)
                         E      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:784)
                         E      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:551)
                         E      at dalvik.system.NativeStart.main(Native Method)
                         E  Caused by: de.greenrobot.event.EventBusException: Subscriber class org.wordpress.android.ui.main.MeFragment already registered to event class org.wordpress.android.ui.prefs.PrefsEvents$MyProfileDetailsChanged
                         E      at de.greenrobot.event.EventBus.subscribe(EventBus.java:179)
                         E      at de.greenrobot.event.EventBus.register(EventBus.java:165)
                         E      at de.greenrobot.event.EventBus.register(EventBus.java:133)
                         E      at org.wordpress.android.ui.main.MeFragment.onResume(MeFragment.java:133)
                         E      at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:873)
                         E      at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1032)
                         E      at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1014)
                         E      at android.app.FragmentManagerImpl.dispatchResume(FragmentManager.java:1776)
                         E      at android.app.Activity.performResume(Activity.java:4549)
                         E      at android.app.ActivityThread.performResumeActivity(ActivityThread.java:2434)
                         E      ... 10 more
```

--- a/WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java
@@ -121,6 +121,12 @@ public class MeFragment extends Fragment {
     }
 
     @Override
+    public void onStart() {
+        super.onStart();
+        EventBus.getDefault().register(this);
+    }
+
+    @Override
     public void onStop() {
         EventBus.getDefault().unregister(this);
         super.onStop();
@@ -130,7 +136,6 @@ public class MeFragment extends Fragment {
     public void onResume() {
         super.onResume();
         refreshAccountDetails();
-        EventBus.getDefault().register(this);
     }
 
     @Override

